src/util/cmdlineargs.cpp

```diff
@@ -170,18 +169,6 @@ bool CmdlineArgs::parse(const QStringList& arguments, CmdlineArgs::ParseMode mod
     parser.addOption(timelinePath);
     parser.addOption(timelinePathDeprecated);
 
-    const QCommandLineOption controllerDebug(QStringLiteral("controller-debug"),
-            forUserFeedback ? QCoreApplication::translate("CmdlineArgs",
-                                      "Causes Mixxx to display/log all of the controller data it "
-                                      "receives and script functions it loads")
-                            : QString());
-    QCommandLineOption controllerDebugDeprecated(
-            QStringList({QStringLiteral("controllerDebug"),
-                    QStringLiteral("midiDebug")}));
-    controllerDebugDeprecated.setFlags(QCommandLineOption::HiddenFromHelp);
-    parser.addOption(controllerDebug);
-    parser.addOption(controllerDebugDeprecated);
```

Comment:
Didn't we agree that we would still check for the controllerDebug option and print a suitable logging rule string as a replacement?

Reply:
No.

Reply:
I've already [said](https://github.com/mixxxdj/mixxx/pull/4239#issuecomment-908069608) why not [twice](https://github.com/mixxxdj/mixxx/pull/4239#issuecomment-955522443). There's nothing more to discuss. Merge or don't.

Reply:
I'm not suggesting to emulate the old behavior. I just suggested to print a hint for inexperienced users. See this comment: https://github.com/mixxxdj/mixxx/pull/4239#issuecomment-955448051

Reply:
I'm not doing that.

Reply:
That attitude is not helpful and you know that. Please explain your reasoning, this attitude is incredibly frustrating to work with.

Reply:
This code was done months ago.

Reply:
Yes and it has not been merged because we could not find a consensus to bring it into a state everybody was comfortable merging it in. My suggestion was intended to compromise so we can find a consensus with minimal extra effort, yet you still refuse to accept a compromise without explaining yourself.
I'm open to discussion, but refusing any discussion is even more counterproductive.

Reply:
I have no obligation to repeat myself again.

Reply:
Then at least link me to where you initially explained why you refuse to acknowledge this suggestion. Please